mandatory/2-function-creation.js

```diff
@@ -6,17 +6,33 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num === "number" && num %2 === 0 && num <= 100)
```

Comment:
That's great. We also can use another way.
return (typeof num == "number"  &&  num%2 === 0  &&  num <= 100 );
it will be return true if all 3 condition are true  ;